apps/nextjs/src/app/aila/help/index.tsx

```diff
@@ -30,196 +44,247 @@ export const HelpContent = () => {
+                      onClick={() => scrollToRefWithOffset(downloadingRef)}
+                    >
+                      Downloading and sharing
+                    </OakLink>
+                  </OakLI>
+                  <OakLI $pv="inner-padding-xs">
+                    <OakLink onClick={() => scrollToRefWithOffset(aiRef)}>
+                      AI and accuracy
+                    </OakLink>
+                  </OakLI>
+                </OakUL>
+              </OakBox>
+            </OakBox>
+          </OakGridArea>
+          <OakGridArea $colStart={[0, 0, 5]} $colSpan={[12, 12, 9]}>
+            <OakFlex
+              $flexDirection={"column"}
+              $position={"relative"}
+              $gap={"space-between-m2"}
+            >
+              <OakBox $display={["block", "none", "none"]}>
+                <OakLink
+                  href={ailaId ? `/aila/${ailaId}` : "/aila"}
+                  iconName="chevron-left"
                 >
-                  Starting your first lesson
-                </button>
-              </li>
-              <li>
-                <button
-                  className="mb-14 text-blue underline"
-                  onClick={() => scrollToRefWithOffset(structureRef)}
+                  Back to Aila
+                </OakLink>
+              </OakBox>
+
+              <OakBox $mb={"space-between-xs"}>
+                <GetInTouchBox />
+              </OakBox>
+              <OakBox ref={startingRef}>
+                <OakHeading
+                  $mb={"space-between-ssx"}
+                  $font="heading-5"
+                  tag="h2"
                 >
+                  Starting your first lesson
+                </OakHeading>
+                <OakP>
+                  Create your first lesson by typing a lesson title and key
+                  stage into the input box on the first screen and clicking the
+                  arrow key. This will create the starting point of your lesson.
+                  You can get back to the initial screen by clicking &apos;New
+                  lesson&apos; in the main navigation.
+                </OakP>
+              </OakBox>
+              <OakBox ref={structureRef}>
+                <OakHeading $mb={"space-between-xs"} $font="heading-5" tag="h2">
                   Structure of an Oak lesson
-                </button>
-              </li>
-              <li>
-                <button
-                  className="mb-14 text-blue underline"
-                  onClick={() => scrollToRefWithOffset(downloadsRef)}
-                >
+                </OakHeading>
+                <OakP $mb={"space-between-xs"}>
+                  Lessons are created using Oak&apos;s curriculum principles and
+                  include the following sections:
+                </OakP>
+                <OakUL className="mb-6 list-disc pl-12">
+                  <OakLI>Learning outcome</OakLI>
+                  <OakLI>Learning cycle outcomes</OakLI>
```

Comment:
This seems different to the language in the app

Reply:
Yeah good spot, have adjusted to match current lesson sections